librz/core/cmd/cmd_print.c

```diff
@@ -5575,509 +5540,12 @@ RZ_IPI int rz_cmd_print(void *data, const char *input) {
 	}
 		rz_cons_break_push(NULL, NULL);
 		switch (input[1]) {
-		case 'j': // "pxj"
-			rz_core_print_jsondump(core, core->block, core->blocksize, 8);
-			break;
 		case '/': // "px/"
 			rz_core_print_examine(core, input + 2);
```

Comment:
This wasn't converted, it seems?

Reply:
Yes, it's too flexible, so keep it as `RZ_CMD_DESC_TYPE_OLDINPUT`

```
[0x00005ef0]> px/?
Format is x/[num][format][size]
Num specifies the number of format elements to display
Format letters are o(octal), x(hex), d(decimal), u(unsigned decimal),
  t(binary), f(float), a(address), i(instruction), c(char) and s(string),
  T(OSType), A(floating point values in hex).
Size letters are b(byte), h(halfword), w(word), g(giant, 8 bytes).

```

Reply:
@ret2libc what do you think?

Reply:
i like it, much more flexible

Reply:
usually `giant` is `double-word`

Reply:
```
[0x00005ef0]> x?
Usage: px[0afoswqWqQ][f]   # Print heXadecimal
| px                show hexdump
| px/               same as x/ in gdb (help x)
| px0               8bit hexpair list of bytes until zero byte
| pxa               show annotated hexdump
| pxA[?]            show op analysis color map
| pxb               dump bits in hexdump form
| pxc               show hexdump with comments
| pxd[?1248]        signed integer dump (1 byte, 2 and 4)
| pxe               emoji hexdump! :)
| pxf               show hexdump of current function
| pxh               show hexadecimal half-words dump (16bit)
| pxH               same as above, but one per line
| pxi               HexII compact binary representation
| pxl               display N lines (rows) of hexdump
| pxo               show octal dump
| pxq               show hexadecimal quad-words dump (64bit)
| pxQ[q]            same as above, but one per line
| pxr[1248][qj]     show hexword references (q=quiet, j=json)
| pxs               show hexadecimal in sparse mode
| pxt[*.] [origin]  show delta pointer table in rizin commands
| pxw               show hexadecimal words dump (32bit)
| pxW[q]            same as above, but one per line (q=quiet)
| pxx               show N bytes of hex-less hexdump
| pxX               show N words of hex-less hexdump
[0x00005ef0]> x/
[0x00005ef0]> x/?
Format is x/[num][format][size]
Num specifies the number of format elements to display
Format letters are o(octal), x(hex), d(decimal), u(unsigned decimal),
  t(binary), f(float), a(address), i(instruction), c(char) and s(string),
  T(OSType), A(floating point values in hex).
Size letters are b(byte), h(halfword), w(word), g(giant, 8 bytes).
[0x00005ef0]> x/10fb
0x00005ef0 = -2.05176598e+35
0x00005ef4 = -2.43060218e-33
0x00005ef8 = -2.41187068e-33
0x00005efc = -1.93742019e+22
0x00005f00 = 3397.05859
0x00005f04 = -728067.062
0x00005f08 = 5.08824453e+10
0x00005f0c = -nan
0x00005f10 = 6.96386875e-38
0x00005f14 = 5.25126609e-11
```

It looks OK

Reply:
I added entry about it to https://github.com/rizinorg/rizin/issues/1590, to not forget